### ISSUE TITLE
error[E0308]: mismatched types

### ISSUE THREAD

**Issue by fbordwell** [TsuITOAR/visa-rs#3]
Building in both versions 0.4.0 and 0.5.0 gives the following error on Ubuntu 22.04:
```rust
error[E0308]: mismatched types
  --> /home/fbordwell/.cargo/git/checkouts/visa-rs-657bd5cc4c91eb01/a805029/src/flags.rs:12:25
   |
12 |         const NO_LOCK = vs::VI_NO_LOCK;
   |                         ^^^^^^^^^^^^^^ expected `u64`, found `u32`

error[E0308]: mismatched types
  --> /home/fbordwell/.cargo/git/checkouts/visa-rs-657bd5cc4c91eb01/a805029/src/flags.rs:13:32
   |
13 |         const EXCLUSIVE_LOCK = vs::VI_EXCLUSIVE_LOCK;
   |                                ^^^^^^^^^^^^^^^^^^^^^ expected `u64`, found `u32`

error[E0308]: mismatched types
  --> /home/fbordwell/.cargo/git/checkouts/visa-rs-657bd5cc4c91eb01/a805029/src/flags.rs:14:29
   |
14 |         const SHARED_LOCK = vs::VI_SHARED_LOCK;
   |                             ^^^^^^^^^^^^^^^^^^ expected `u64`, found `u32`

error[E0308]: mismatched types
  --> /home/fbordwell/.cargo/git/checkouts/visa-rs-657bd5cc4c91eb01/a805029/src/flags.rs:15:29
   |
15 |         const LOAD_CONFIG = vs::VI_LOAD_CONFIG;
   |                             ^^^^^^^^^^^^^^^^^^ expected `u64`, found `u32`

For more information about this error, try `rustc --explain E0308`.
error: could not compile `visa-rs` due to 4 previous errors
```

**Comment by fbordwell** [comment]
Platform issue, fixed by rebinding VISA library.

**Comment by TsuITOAR** [comment]
Sorry I haven't checked this repository for a while, this should be fixed by visa-sys version 0.1.5
